src/main/java/team/themoment/hellogsmv3/domain/oneseo/dto/response/OneseoPrivacyDetailResDto.java

```diff
@@ -31,6 +31,7 @@ public record OneseoPrivacyDetailResDto(
         String schoolAddress,
         String schoolTeacherName,
         String schoolTeacherPhoneNumber,
-        String profileImg
+        String profileImg,
+        LocalDate graduationDate
```

Comment:
```suggestion
        String profileImg,
        @JsonSerialize(using = LocalDateSerializer.class)
        @JsonDeserialize(using = LocalDateDeserializer.class)
        @JsonFormat(shape= JsonFormat.Shape.STRING, pattern="yyyy-MM-dd")
        LocalDate graduationDate
```

테스트해보았는데 java 8버전부터 생긴 LocalDate 타입은 jackson 라이브러리에서 역직렬화 할 수 없어서 500에러가 발생하는것으로 확인됩니다.
이를 해결하려면 위의 birth 필드와 마찬가지로 위와 같은 어노테이션을 필드에 붙혀 해결해야할 것 같습니다.